packages/webdriver/src/driver/index.ts

```diff
@@ -85,17 +86,33 @@ export async function startWebDriver (options: Options.WebDriver) {
             buildId,
             cacheDir
         })
+        let loggedBuildId = buildId
         const hasChromedriverInstalled = await fsp.access(chromedriverBinaryPath).then(() => true, () => false)
         if (!hasChromedriverInstalled) {
             log.info(`Downloading Chromedriver v${buildId}`)
-            await install({
+            const chromedriverInstallOpts: InstallOptions & {unpack?: true} = {
                 ...chromedriverOptions,
                 cacheDir,
                 platform,
                 buildId,
                 browser: Browser.CHROMEDRIVER,
                 unpack: true,
                 downloadProgressCallback: (downloadedBytes, totalBytes) => downloadProgressCallback('Chromedriver', downloadedBytes, totalBytes)
+            }
+            await install({ ...chromedriverInstallOpts, buildId }).catch(async (err) => {
+                /**
+                 * in case we detect a Chrome browser installed for which there is no Chromedriver available
+                 * we are falling back to the latest known good version
+                 */
+                log.warn(`Couldn't download Chromedriver v${buildId}: ${err.message}, trying to find known good version...`)
+                const majorVersion = buildId.split('.')[0]
+                const knownGoodVersions: any = await got('https://googlechromelabs.github.io/chrome-for-testing/known-good-versions.json').json()
+                const knownGoodVersion = knownGoodVersions.versions.filter(({ version }: { version: string }) => version.startsWith(majorVersion)).pop()
+                if (!knownGoodVersion) {
+                    throw new Error(`Couldn't find known good version for Chromedriver v${majorVersion}`)
+                }
+                loggedBuildId = knownGoodVersion.version
+                return install({ ...chromedriverInstallOpts, buildId: loggedBuildId })
```

Comment:
Can we have a case when known good version has already been installed and can be used from cache?

Reply:
So this way, console won't contain message about the version of Chromedriver in use, right?
Like we have for else flow in row 118